scripts/generateSiteFiles.R

```diff
@@ -0,0 +1,95 @@
+#Title: generate SiteFiles
+# Based on the generateSiteFiles.R from Bora Uyar : https://github.com/BIMSBbioinfo/crispr_DART/blob/master/src/generateSiteFiles.R
+#not changed yet
+# the target regions would be samples in our case
+#which params are needed ? ( samples, dir_reportfiles, dir_output ... )
+
+
+library(data.table)
+library(yaml)
+
+args = commandArgs(trailingOnly=TRUE)
+
+reportsScriptDir <- args[1] #folder that contains the Rmd reports that will be rendered into a site
+sampleSheetFile <- args[2] #path to sample_sheet.csv file
+cutSitesFile <- args[3] #path to cut_sites.bed file
+comparisonsFile <- args[4] #path to tsv file containing list of pairs of samples to compare 
+pipelineOutputDir <- args[5] #root folder where the pipeline is written to
+siteDir <- args[6] #path to folder where the site will be generated
+
+#read sample sheet
+sampleSheet <- data.table::fread(sampleSheetFile)
+
+#1. for each target region in sample sheet, create a copy of all Rmd files
+#replacing the target_name parameter in the Rmd file's yaml header.
+rmd_files <- dir(reportsScriptDir, pattern = '.Rmd$', full.names = T)
+
+for(f in rmd_files) {
+  if(basename(f) == 'index.Rmd') {
+    file.copy(from = f, to = siteDir)
+  } else if (grepl('^_.+.Rmd$', basename(f))){ #copy child Rmd files don't duplicate 
+    file.copy(from = f, to = siteDir)
+  } else {
+    text <- readLines(f)
+    for(target_name in unique(sampleSheet$target_name)) {
+      outFile <- file.path(siteDir, paste0(target_name, ".", basename(f)))
+      modified_text <- gsub(x = text,
+                            pattern = ' @PARAMS_TARGET_NAME@:',
+                            replacement = paste0("target_name: '", target_name, "'"))
+      writeLines(text = modified_text, con = outFile)
+    }
+  }
+}
```

Comment:
As far as I can tell, this code is not actually used, or is it?  There is no PARAMS_TARGET_NAME anywhere to be found.

What is the intention here?